assets/css/pre_fare_v2.scss

```diff
@@ -29,6 +29,10 @@
 @import "v2/pre_fare/elevator_status";
 @import "v2/pre_fare/flex/paging_indicator";
 
+// Can this scss be generic?
```

Comment:
I know this comment is probably for the future, but I think yes. There shouldn't be much (if any) differences between how Bus Shelter and Pre-Fare display Subway Status.

Reply:
I might as well do that now!